.gitignore

```diff
@@ -2,3 +2,4 @@
 .pub
 packages
 pubspec.lock
+.idea
```

Comment:
Please, no.  We should have folks ignore these globally!

Reply:
👍